jobs/v4/howto/create_job_sample.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+// [START job_search_create_job]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	talent "cloud.google.com/go/talent/apiv4beta1"
+	talentpb "google.golang.org/genproto/googleapis/cloud/talent/v4beta1"
+)
+
+// createJob create a job as given.
+func createJob(w io.Writer, projectID, companyName, requisitionID, title, URI, description, address1, address2, languageCode string) (*talentpb.Job, error) {
```

Comment:
Oh! Because you use these from the TestMain setup!

It wouldn't hurt to add an explicit test file, but it's also OK to leave it as-is because it's tested. (Although the STDOUT of createJob isn't actually tested... which would be good to check)